e2e/test/scenarios/visualizations/pivot_tables.cy.spec.js

```diff
@@ -1051,6 +1052,29 @@ describe("scenarios > visualizations > pivot tables", { tags: "@slow" }, () => {
       });
     });
   });
+
+  it("should not have to wait for data to show fields in summarisation (#26467)", () => {
+    createAndVisitTestQuestion();
+
+    cy.intercept("POST", "api/card/pivot/*/query", req => {
+      req.on("response", res => {
+        res.setDelay(20_000);
+      });
+    }).as("querySlowed");
+
+    cy.reload();
```

Comment:
Reloading the page is one of the most time-consuming operations in Cypress.
Can we remove it and just move `createAndVisitTestQuestion()` here, after the intercept?

Reply:
Good point thanks. To avoid the auto wait of the query (done in cy.question) I changed it to manually visit the question and I had to do some edits in the helper function, but hopefully the test is faster now.
Let me know if you know of a better way of doing that

Reply:
Looks good, thanks for leaving a comment!